staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -405,6 +421,57 @@ func (t *tracker) add(gvr schema.GroupVersionResource, obj runtime.Object, ns st
 	return nil
 }
 
+type objectVersioner struct{}
+
+func (a objectVersioner) objectResourceVersion(obj runtime.Object) (uint64, error) {
+	accessor, err := meta.Accessor(obj)
+	if err != nil {
+		return 0, err
+	}
+	version := accessor.GetResourceVersion()
+	if len(version) == 0 {
+		return 0, nil
+	}
+	return strconv.ParseUint(version, 10, 64)
+}
+
+// CompareResourceVersion compares resource versions.
+func (a objectVersioner) CompareResourceVersion(lhs, rhs runtime.Object) int {
+	lhsVersion, err := a.objectResourceVersion(lhs)
+	if err != nil {
+		// coder error
+		panic(err)
+	}
+	rhsVersion, err := a.objectResourceVersion(rhs)
+	if err != nil {
+		// coder error
+		panic(err)
+	}
+
+	if lhsVersion == rhsVersion {
```

Comment:
return (lhsVersion - rhsVersion) % 2 

(if you want to take it)

Reply:
Thanks! But I let it be for now, because current `objectVersioner` may be replaced by better solution.